en/docs/learn/logging-into-drupal-using-the-identity-server.md

```diff
@@ -0,0 +1,202 @@
+# Logging into Drupal using the Identity Server
+
+Drupal is open source content management software distributed under the terms of the[ GNU General Public License](http://www.gnu.org/copyleft/gpl.html) (GPL). This topic provides instructions on how to log into Drupal using your WSO2 Identity Server credentials. In this tutorial, Drupal acts as the service provider and WSO2 Identity Server acts as the identity provider.
+
+Let's get started!
+
+!!! tip "Before you begin!"
+    You need to have Drupal installed. Click[ here](https://www.drupal.org/docs/installing-drupal) for more information on installing Drupal.
+    You need to have Composer installed. Click[ here](https://getcomposer.org/) for more information on installing Composer.
+    You need to have SimpleSAMLphp Authentication Drupal modules installed. This module helps Drupal to communicate with WSO2 Identity Server. You can install it using the composer by executing the following command.
+    
+    ```
+    composer require drupal/simplesamlphp_auth
+    ```
+## Configure SimpleSAMLphp as Service Provider.
+
+1.  Navigate to the Web directory on a terminal window and run the following command to create a symlink to the **vendor/simplesamlphp/simplesamlphp/www** folder.
```

Comment:
Please use monospace instead of bold for paths. 
`vendor/simplesamlphp/simplesamlphp/www`